components/commons/overlay-coretokens.css

```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2019 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+%spectrum-overlay {
+  /* TODO: replace with core tokens */
+  --spectrum-overlay-animation-distance: 6px;
```

Comment:
Agree, can we capture those please in this Paperdoc so we keep track https://paper.dropbox.com/doc/Missing-tokens-from-Core-Tokens-CSS-Migration-y9o9A3tt2qKFBSzUjfhsY

Reply:
Done! That 6px distance was set by Garth, I believe. He did some intervention to help me get overlay to import into popover. 

In the case of Popover, the distance may need to be 8px to achieve the correct popover distance to source (a token deemed out of scope for popover). I've noted that in the document.